code/drasil-example/Drasil/GamePhysics/DataDefs.hs

```diff
@@ -43,7 +43,7 @@ ctrOfMass = mkQuantDef posCM ctrOfMassEqn
 
 -- FIXME (Variable "i") is a horrible hack
 ctrOfMassEqn :: Expr
-ctrOfMassEqn = sumAll (Variable "i") (sy massI * sy posI) / sy mTot
+ctrOfMassEqn = sumAll (Variable "i") (sy massj * sy posj) / sy mTot
```

Comment:
Hmm, something here seems amiss. The hack here is likely quite wrong. It does look like `mass` and `pos` depend on `j` (as defined in `Unitals`), but this sum over `i` is then wrong. Could you check in the generated PDF and HTML if that's indeed the case?